tls/s2n_connection.c

```diff
@@ -346,8 +346,8 @@ int s2n_connection_set_blinding(struct s2n_connection *conn, s2n_blinding blindi
 
 int s2n_connection_get_delay(struct s2n_connection *conn)
 {
-    /* Delay between 1ms and 10 seconds */
-    return s2n_public_random(1000 + 10000000);
+    /* Delay between 1ms and 10 seconds in microseconds */
+    return 1000 + s2n_public_random(10 * 1000 * 1000 - 1000);
```

Comment:
Thanks for this fix! Would you mind making the operator precedence explicit with parens? e.g. (10 * 1000 * 1000) - 1000 ?